lib/buildArtifacts/artifacts-public-access.ts

```diff
@@ -31,7 +31,7 @@ export class ArtifactsPublicAccess {
     }));
 
     const urlRewriter = new NodejsFunction(stack, 'CfUrlRewriter', {
-      runtime: Runtime.NODEJS_14_X,
+      runtime: Runtime.NODEJS_16_X,
```

Comment:
node16 is already eol and consider it is very similar with node18, can we just move to node18 here?